great_expectations/render/renderer/v3/suite_profile_notebook_renderer.py

```diff
@@ -105,7 +105,7 @@ def add_footer(
     ]
 }
 checkpoint = SimpleCheckpoint(
-    f"_tmp_checkpoint_{expectation_suite_name}",
+    f"{validator.active_batch_definition.data_asset_name}_{expectation_suite_name}",
```

Comment:
Using `validator.active_batch_definition.data_asset_name` to retrieve the asset name since the validator is already configured in the notebooks.